forge-game/src/main/java/forge/game/card/CardFactoryUtil.java

```diff
@@ -4021,9 +4060,12 @@ public static void setFaceDownState(Card c, SpellAbility sa) {
         if (sa.hasParam("FaceDownSetType")) {
             faceDown.setType(new CardType(Arrays.asList(sa.getParam("FaceDownSetType").split(" & ")), false));
         }
+        if (sa.hasParam("FaceDownKeyword")) {
+            faceDown.addIntrinsicKeywords(Arrays.asList(sa.getParam("FaceDownKeyword").split(" & ")));
```

Comment:
shouldn't the spell already have Ward?
I copied the disguised Phantom with _Lithoform Engine_ and the token didn't have it 🤔

Reply:
should be updated, i think

Reply:
the token is still without it

Reply:
then its something else broken

the Token Spell on the Stack still has ward, but on the battlefield it has not

maybe the GameAction does something bad with the copy?

Reply:
it seems the Spell does have Ward on the Stack but the copied Card host doesn't ? or i'm seeing this wrong?

can you debug this?

Reply:
the `abilityCastFaceDown.resolve()` needs better logic so it doesn't turn the copy face down

Reply:
fixed i think